pkgs/development/python-modules/paddlepaddle/default.nix

```
@@ -0,0 +1,57 @@
+{ stdenv
+, lib
+, buildPythonPackage
+, fetchPypi
+, python
+, pythonOlder
+, pythonAtLeast
+# runtime dependencies
+, httpx
+, numpy
+, protobuf
+, pillow
+, decorator
+, astor
+, paddle-bfloat
+, opt-einsum
+}:
+
+let
+  pname = "paddlepaddle";
+  version = "2.5.0";
+  format = "wheel";
+  pyShortVersion = "cp${builtins.replaceStrings ["."] [""] python.pythonVersion}";
+  hashAndPlatform = import ./binary-hashes.nix { pythonVersion = pyShortVersion; system = stdenv.system; };
+  src = fetchPypi ({
+    inherit pname version format;
+    dist = pyShortVersion;
+    python = pyShortVersion;
+    abi = pyShortVersion;
+  } // hashAndPlatform);
+in
+buildPythonPackage {
+  inherit pname version format src;
+
+  disabled = pythonOlder "3.9" || pythonAtLeast "3.11";
+
```

```suggestion

  nativeBuildInputs = [
    autoPatchelfHook
  ];
  
  buildInputs = [
    openssl_1_1
    zlib
  ];
  
```

We can use `autoPatchelfHook` here.
However, openssl_1_1 is already deprecated in nixpkgs, so we need to do something about it.